server/src/main/java/eu/fasten/server/FastenServer.java

```diff
@@ -54,7 +51,17 @@
             defaultValue = ".")
     private Path pluginPath;
 
-    @Option(names = {"-t", "--topic"},
+    @Option(names = {"-s", "--skip_offsets"},
+            paramLabel = "skip",
+            description = "Adds one to offset of all the partitions of the consumers.",
+            defaultValue = "0")
+    private int skipOffsets;
+
+    @Option(names = {"-m", "--mode"},
+            description = "Deployment or Development mode")
+    private boolean deployMode;
+
+    @Option(names = {"-kt", "--topic"},
             paramLabel = "topic",
             description = "JSON file consists of topics for plug-ins.")
```

Comment:
What does this mean?

Reply:
Give default value and perhaps example

Reply:
Actually, why does this need to be a JSON string? JSON is very difficult to manipulate at the command line. Can we just provide a list of KV pairs, e.g. PLUGIN1:TOPIC_NAME1, PLUGIN2:TOPIC_NAME2 or even better allow `-kt` to repeat?

Reply:
I implemented key-value pairs, but instead of `:` I'm using `=` which is a way `picocli` is handling maps, which seems like a natural way of handling things such as this one. So we'll have OPAL=fasten.maven.pkg for example.